src/Features/CSharp/Portable/Completion/CompletionProviders/CrefCompletionProvider.cs

```diff
@@ -42,6 +42,16 @@ internal sealed class CrefCompletionProvider : AbstractCrefCompletionProvider
                     SymbolDisplayMiscellaneousOptions.EscapeKeywordIdentifiers |
                     SymbolDisplayMiscellaneousOptions.UseSpecialTypes);
 
+        public static readonly SymbolDisplayFormat SpecialTypeCrefFormat =
+            new SymbolDisplayFormat(
+                globalNamespaceStyle: SymbolDisplayGlobalNamespaceStyle.Omitted,
+                typeQualificationStyle: SymbolDisplayTypeQualificationStyle.NameOnly,
+                propertyStyle: SymbolDisplayPropertyStyle.NameOnly,
+                genericsOptions: SymbolDisplayGenericsOptions.IncludeTypeParameters,
+                parameterOptions: SymbolDisplayParameterOptions.None,
+                miscellaneousOptions:
+                    SymbolDisplayMiscellaneousOptions.EscapeKeywordIdentifiers);
```

Comment:
The naming confuses me. The one called "SpecialTypeCrefFormat..." omits the UseSpecialTypes flag, but the non-special "CrefFormat" includes UseSpecialTypes. I get that the point is to make a non-special item for what would otherwise be a special type (thanks to an offline discussion), but I'd update the name to be more clear.